osidb/tests/factories.py

```diff
@@ -108,7 +122,8 @@ def _create(cls, model_class, *args, **kwargs):
         # but we want preserve it as writable factory attribute as it is easier to work with
         # than with ACLs so we need to remove it for the flaw creation and emulate annotation
         embargoed = kwargs.pop("embargoed")
-        flaw = super()._create(model_class, *args, **kwargs)
+        flaw = model_class(*args, **kwargs)
+        flaw.save(raise_validation_error=False)
```

Comment:
I replaced `super()._create(model_class, *args, **kwargs)` because it call a save method with validation.
Being both Affect and Flaw dependent on each other I have to first save one without validation to generate an UUID and then be able to associate each other, if I don´t generate this UUID first, when the `RelatedFactoryList` is called to generate the affect (after the entire creation of the flaw) the database will raise error due the foreign key not existing.

Also it call a save method again with the validations, (I believe it is in the postgeneration method I wasn't able to find where in the docs) but if the `FlawFactory` is passed with an validation error like `FlawFactory(embargoed=True, title="test")`, even skiping validations in `_create` method we raise the validation error `Flaw title does not contains "EMBARGOED" despite being embargoed.`